### PR TITLE
Documentation general updates

### PR DESCRIPTION
**Pull request by carolinamenezes**
#### How to test it?

[Workspace](Link goes here!)

### CODE REVIEW

**Review by Mariana-Caetano — APPROVED** [review]
Awesome! 🚀

**Review by Mariana-Caetano — APPROVED** [review]

File: docs/AvailabilitySubscriber.md [hunk 1]
```diff
@@ -1,99 +1,69 @@
...
-| `content` | `AvailabilitySubscriber` form content container | [index](/react/components/AvailabilitySubscriber/index.js) |
-| `input` | `AvailabilitySubscriber` form input containter | [index](/react/components/AvailabilitySubscriber/index.js) |
-| `inputName` | `AvailabilitySubscriber` form input name | [index](/react/components/AvailabilitySubscriber/index.js) |
-| `inputEmail` | `AvailabilitySubscriber` form input email | [index](/react/components/AvailabilitySubscriber/index.js) |
-| `submit` | `AvailabilitySubscriber` form submit button | [index](/react/components/AvailabilitySubscriber/index.js) |
-| `success` | `AvailabilitySubscriber` success feedback message | [index](/react/components/AvailabilitySubscriber/index.js) |
-| `error` | `AvailabilitySubscriber` error feedback message | [index](/react/components/AvailabilitySubscriber/index.js) |
+For now this block does not have any mandatory or optional blocks.
+
+#### Customization
```
Comment (carolinamenezes) [inline docs/AvailabilitySubscriber.md:53]:
```suggestion
## Customization
```

File: docs/BackToTopButton.md [hunk 2]
```diff
@@ -47,17 +47,19 @@ _Back To Top Button rendered as a caret icon_
...
 
+### Props
+
 | Prop name          | Type     | Description                                                                                                                                                 | Default Value |
 | ------------------ | -------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
-| `displayThreshold` | `number` | Defines the window Y pixel in which the button will be displayed.                                                                                           | `600`         |
 | `display`          | `enum`   | Defines the component rendering. Possible values are: `button` to display a button with a `Back To Top` label text or `caret-icon` to display just an icon. | `button`      |
+| `displayThreshold` | `number` | Defines the window Y pixel in which the button will be displayed.                                                                                           | `600`         |
 
 ### Customization
```
Comment (carolinamenezes) [inline docs/BackToTopButton.md:57]:
```suggestion
## Customization
```

File: docs/ProductImages.md [hunk 3]
```diff
@@ -100,22 +106,22 @@ Check out an example below:
...
 | `defaultSize` | `number` |  Image default size (in `px`). | `1200` | 
 | `imageSizes` | `[number]` | Image size(s) (in `px`) to be used in the image's [`srcset` HTML attribute](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images). If no value is passed to this prop, the `srcset` will use the image original size.  | `undefined` |
 | `maxSize` | `number` | Image maximum size (in `px`) for rendering regardless of the screen size. Notice that this prop only works if you also declare the `imageSizes` prop. | `4096` |
+| `zoomFactor` | `number` | Sets how much the zoom increases the image size (e.g. `2` will make the zoomed-in image twice as large). | `2` |
+| `zoomMode` | `enum` | Defines the zoom behavior for the `product-images.high-quality-image` block. Possible values are: `disabled` (zoom is disabled), `in-place-click`(zoom will be triggered when the image is clicked on), or `in-place-hover`(zoom will be triggered when the image is hovered on). Different from the `store-images` prop, this one doesn't accept `open-modal` value. | `disabled` |
 
 #### Customization
```
Comment (carolinamenezes) [inline docs/ProductImages.md:122]:
```suggestion
## Customization
```

File: docs/SearchBar.md [hunk 4]
```diff
@@ -1,79 +1,86 @@
...
 | `clear-button`             | `'clear-button'`             | ![clear-button-empty](https://user-images.githubusercontent.com/12702016/74764904-5cc5e580-5261-11ea-9df8-484cf457c266.png)             | ![clear-button-filled](https://user-images.githubusercontent.com/12702016/74764917-60f20300-5261-11ea-8911-11c8fd9582d9.png)             |
 | `search-and-clear-buttons`            | `'search-and-clear-buttons'`            | ![search-and-clear-buttons-empty](https://user-images.githubusercontent.com/12702016/74764924-62bbc680-5261-11ea-9f1d-2118274da996.png) | ![search-and-clear-buttons-filled](https://user-images.githubusercontent.com/12702016/74764928-64858a00-5261-11ea-9ed2-42da887e6641.png) |
 | `search-button` | `'search-button'` | ![search-button-empty](https://user-images.githubusercontent.com/12702016/74764929-65b6b700-5261-11ea-815c-ecc9f0c44e0f.png)            | ![search-button-filled](https://user-images.githubusercontent.com/12702016/74764934-66e7e400-5261-11ea-8a86-59da9a1c0faa.png)            |
 
-### `HorizontalAlignment`
+### Customization
```
Comment (carolinamenezes) [inline docs/SearchBar.md:69]:
```suggestion
## Customization
```

File: docs/Share.md [hunk 5]
```diff
@@ -27,58 +25,49 @@ In the following sections, learn how to configurate the component in your store.
...
-| `Twitter` | `Boolean` | If twitter social media will be shown |
-| `WhatsApp` | `Boolean` | If whatsApp social media will be shown |
-
-### Styles API
-You should follow the Styles API instruction in [Store Components Styles API](https://github.com/vtex-apps/store-components#styles-api).
+| `Facebook` | `Boolean` | If Facebook social media will be shown. |
+| `Twitter` | `Boolean` | If Twitter social media will be shown. |
+| `WhatsApp` | `Boolean` | If WhatsApp social media will be shown. |
 
 ### Customization
```
Comment (carolinamenezes) [inline docs/Share.md:62]:
```suggestion
## Customization
```

**Review by igorbrasileiro — APPROVED** [review]

**Pull request merged**